pkg/subctl/cmd/gather/tally.go

```diff
@@ -0,0 +1,356 @@
+    <tr>
+      <td>{{.Name}}</td>
+      <td style="width: 40%;">{{.Info.OperatingSystem}} {{.Info.OSImage}} {{.Info.KernelVersion}} {{.Info.Architecture}}</td>
+      <td style="width: 25%;">{{.Info.ContainerRuntimeVersion}}</td>
+      <td style="width: 15%;">{{.Info.KubeletVersion}}</td>
+      <td style="width: 15%;">{{.Info.KubeProxyVersion}}</td>
+    </tr>
+  {{end}}
+</table>
+<h3 id="log-collected"><a href="#{{.ClusterName}}-log-collected">Summary of logs gathered</a></h3>
+{{ if .PodLogs }}
+  <table style="table-layout:fixed; width:100%; text-align:left;">
+    <tr>
+      <th>Pod name</th>
+      <th>Namespace</th>
+      <th>Pod state</th>
+      <th>Restart count</th>
+      <th>Log file(s)</th>
+    </tr>
+    {{range .PodLogs}}
+      <tr>
+        <td style="width: 5%;">{{.PodName}}</td>
+        <td style="width: 5%;">{{.Namespace}}</td>
+        <td style="width: 5%;">{{.PodState}}</td>
+        {{ if gt .RestartCount 0 }}
+          <td style="width: 5%; color: red">{{.RestartCount}}</td>
+        {{ else }}
+          <td style="width: 5%;">{{.RestartCount}}</td>
+        {{end}}
+        {{range $index, $filename := .LogFileName}}
+          <td style="width: 40%;"><a href="{{$filename}}" target="_blank">{{$filename}}</a></td>
+        {{end}}
+      </tr>
+    {{end}}
+  </table>
+{{ else }}
+  No Logs gathered
+{{end}}
+<h3 id="resources-collected"><a href="#{{.ClusterName}}-resources-collected">Summary of resources gathered</a></h3>
+{{ if .ResourceInfo }}
+  <table style="table-layout:fixed; width:100%; text-align:left;">
+    <tr>
+      <th>Resource name</th>
+      <th>Namespace</th>
+      <th>Resource type</th>
+      <th>Resource file(s)</th>
+    </tr>
+    {{range .ResourceInfo}}
+      <tr>
+        <td style="width: 15%;">{{.Name}}</td>
+        <td style="width: 15%;">{{.Namespace}}</td>
+        <td style="width: 15%;">{{.Type}}</td>
+        <td style="width: 65%;"><a href="{{.FileName}}" target="_blank">{{.FileName}}</a></td>
+      </tr>
+    {{end}}
+  </table>
+{{ else }}
+  No resources gathered
+{{end}}
+</body>
+</html>
+`
+*/
+
+type version struct {
+	Subctl    string
+	Subm      string
+	K8sServer string
+}
+
+type clusterConfig struct {
+	CNIPlugin        string
+	CloudProvider    string
+	TotalNode        int
+	GatewayNode      map[string]types.UID
+	GWNodeNumber     int
+	MasterNode       map[string]types.UID
+	MasterNodeNumber int
+}
+
+type nodeConfig struct {
+	Name string
+	Info v1.NodeSystemInfo
+}
+
+type logInfo struct {
+	PodName      string
+	Namespace    string
+	RestartCount int32
+	PodState     v1.PodPhase
+	LogFileName  []string
+}
+
+type resourceInfo struct {
+	Name      string
+	Namespace string
+	Type      string
+	FileName  string
+}
+
+type data struct {
+	ClusterName   string
+	Versions      version
+	ClusterConfig clusterConfig
+	NodeConfig    []nodeConfig
+	PodLogs       []logInfo
+	ResourceInfo  []resourceInfo
+}
+
+func ClusterInformation(info Info) {
```

Comment:
See  output of `subctl gather` attached as zip in the comment below. `tally.html` is the resultant html file. Probably I should rename it `summary.html`